nni/compression/experiment/config/utils.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import math
+from typing import Any, Optional, Tuple, Dict, List, Type, Union
+
+from torch.nn import Module
+
+from nni.compression.pytorch.utils import count_flops_params
+from .compression import CompressionConfig
+from .vessel import CompressionVessel
+
+
+KEY_MODULE_NAME = 'module_name::'
+KEY_PRUNERS = 'pruners'
+KEY_VESSEL = '_vessel'
+KEY_ORIGINAL_TARGET = '_original_target'
+KEY_THETAS = '_thetas'
+
+
+def sigmoid(x: float, theta0: float = -0.5, theta1: float = 10) -> float:
```

Comment:
sigmoid definitely shouldn't appear here.

Reply:
yes, move it to `trial_entry.py`, and `trial_entry.py` will split to several files in next steps... it's a garbage can right now...